lib/inferno/dsl/fhir_evaluation.rb

```diff
@@ -0,0 +1,87 @@
+require 'dotenv'
+require 'faraday'
+require 'fhir_models'
+require 'thor'
+require 'erb'
+require 'yaml'
+require 'pathname'
+
+require_relative 'fhir_evaluator/config'
+require_relative 'fhir_evaluator/evaluator'
+require_relative 'fhir_evaluator/ig'
+require_relative 'fhir_evaluator/rule'
+require_relative 'fhir_evaluator/evaluation_context'
+require_relative 'fhir_evaluator/evaluation_result'
+require_relative 'fhir_evaluator/data_summary'
+require_relative 'fhir_evaluator/dataset'
+
+module Inferno
+  module DSL
+    module FHIREvaluation
+      class Evaluator
+        def initialize(ig_path, data_path)
+          Dotenv.load
+          Dir.glob(File.join(__dir__, 'fhir_evaluator', 'rules', '*.rb')).each do |file|
```

Comment:
This should not be happening as part of the constructor. It should go with the other requires.

Reply:
Commented out this part for now as we don't read rules at initial integration, will come back at phase 3.